server_apps/data_transfer/Downloads/DownloadFiles.pl

```diff
@@ -73,6 +73,19 @@
   exit -1;
 };
 
+try {
+  system("sed 's/^[ \t]*//;s/[ \t]*$//' < crispr_fasta.fa > crispr_fasta_new.fa");
```

Comment:
You could combine these 3 lines into one with sed's in-place flag "-i".

Example:
`sed -i '' 's/^[ \t]*//;s/[ \t]*$//' crispr_fasta.fa`

https://thoughtbot.com/blog/sed-102-replace-in-place